### DISCUSSION TITLE
Show that `direnv allow` is needed **everytime**

### DISCUSSION THREAD

**Discussion by guettli** [direnv/direnv#1129] [General]
sometimes I overlook the warning, that `direnv allow` is needed.

I would like to get a warning everytime I enter a command in the directory.

Is there a way to achieve that?

**Comment by zimbatm** [comment]
direnv itself doesn't support that.

You could append another hook in $PROMPT_COMMAND that checks `direnv status | gre "Loaded RC allowed false" and prints a warning if it matches. Or change the PS1 accordingly.

**Comment by guettli** [comment]
I found this configuration works fine for https://starship.rs/

`  ~/.config/starship.toml`

```

[custom.direnv]
command = '[[ $(direnv status) =~ "Found RC allowed false" ]] && echo "=========> missing: direnv allow <============"' # shows output of command
detect_files = ['.envrc']
```
